templates/admin/base_admin.html.twig

```diff
@@ -25,8 +25,8 @@
             <li class="nav-item dropdown">
                 <a class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">Category</a>
                 <div class="dropdown-menu">
-                    <a class="dropdown-item" href="{{ path('list_categories') }}">List</a>
-                    <a class="dropdown-item" href="{{ path('new_category') }}">Add</a>
+                    <a class="dropdown-item" href="#">List</a>
+                    <a class="dropdown-item" href="#">Add</a>
```

Comment:
@daria-sieroshtan 
???

Reply:
Navigation will be added in next commit. There is no more categories.